main/webapp/modules/core/scripts/index.js

```diff
@@ -68,24 +68,7 @@ Refine.postCSRF = function(url, data, success, dataType, failCallback) {
    });
 };
 
-var lang = (navigator.language|| navigator.userLanguage).split("-")[0];
-var dictionary = "";
-$.ajax({
-	url : "command/core/load-language?",
-	type : "POST",
-	async : false,
-	data : {
-	  module : "core",
-//		lang : lang
-	},
-	success : function(data) {
-		dictionary = data['dictionary'];
-        lang = data['lang'];
-	}
-});
-$.i18n().load(dictionary, lang);
-$.i18n({ locale: lang });
-// End internationalization
```

Comment:
That change seems to be unrelated to the change of icon, so perhaps it would be cleaner to have it as a separate PR, no?

Reply:
I meant to put that change in a separate PR, I have reverted it and added some slight improvements (see updated screenshot).